app/models/authenticator/httpd.rb

```diff
@@ -42,12 +42,11 @@ def user_authorizable_with_system_token?
     end
 
     def _authenticate(_username, _password, request)
-      request.present? &&
-        request.headers['X-REMOTE-USER'].present?
+      request.present? && request.headers['X-REMOTE-USER'].present?
     end
 
     def failure_reason(_username, request)
-      request.headers['X-EXTERNAL-AUTH-ERROR']
+      request.headers['X-EXTERNAL-AUTH-ERROR']&.force_encoding("UTF-8")
```

Comment:
This modifies the value in the headers in place.  I know this has the side effect that we want but is there anything else expecting this value to be in `ASCII-8BIT`"?

Reply:
In other words, do we need to modify it in place or can we return a copy of it in this method?